pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -471,43 +496,66 @@ func (c *Controller) addNodeRoute(nodeName string, node *corev1.Node) error {
 		}
 		peerGatewayIP := ip.NextIP(peerPodCIDRAddr)
 		peerConfig[peerPodCIDR] = peerGatewayIP
-		podCIDRs = append(podCIDRs, peerPodCIDR)
+		peerPodCIDRs = append(peerPodCIDRs, peerPodCIDR)
 	}
 
-	ipsecTunOFPort := int32(0)
-	if c.networkConfig.EnableIPSecTunnel {
-		// Create a separate tunnel port for the Node, as OVS IPSec monitor needs to
-		// read PSK and remote IP from the Node's tunnel interface to create IPSec
-		// security policies.
-		if ipsecTunOFPort, err = c.createIPSecTunnelPort(nodeName, peerNodeIP); err != nil {
-			return err
-		}
+	peerNodeIPs, err := k8s.GetNodeAddrs(node)
+	if err != nil {
+		klog.Errorf("Failed to retrieve IP addresses of Node %s: %v", nodeName, err)
+		return nil
 	}
 
-	err = c.ofClient.InstallNodeFlows(
-		nodeName,
-		peerConfig,
-		peerNodeIP,
-		uint32(ipsecTunOFPort),
-		peerNodeMAC)
-	if err != nil {
-		return fmt.Errorf("failed to install flows to Node %s: %v", nodeName, err)
+	for _, peerNodeIP := range peerNodeIPs {
```

Comment:
Again, why use Node IP but not transport IP?

Reply:
Updated.